addons/web/static/tests/core/l10n/dates_tests.js

```diff
@@ -724,4 +724,16 @@ QUnit.module("utils", {}, () => {
             "2023-01-22T00:00:00"
         );
     });
+
+    QUnit.test("formatDate, properly formats DateTime.fromISO dates, no matter the TimeZone", async (assert) => {
```

Comment:
I'm not sure this is how you want to test your fix. You should write a tour in the point of sale and make sure the date is correctly displayed. If you need some help writing it don't hesitate to DM on discord

Reply:
This test indeed makes no sense regarding the fix.